src/foam/u2/wizard/agents/DeveloperModeAgent.js

```diff
@@ -30,7 +30,7 @@ foam.CLASS({
       const developerCapability = await this.userCapabilityJunctionDAO.find(
         e.EQ(foam.nanos.crunch.UserCapabilityJunction.TARGET_ID, 'developer'));
 
-      this.developerMode = developerCapability !== undefined;
+      this.developerMode = developerCapability !== null;
```

Comment:
to cover both cases
```suggestion
      this.developerMode = developerCapability != null;
```

Reply:
@KernelDeimos is there any scenario in which developerCapability would be undefined

Reply:
If somebody changes that code or adds a client-side DAO decorator which returns `undefined` it would be difficult for them to notice that they broke this.

Reply:
Ok, that's  fair.